source/evp_fetch.c

```diff
@@ -278,15 +279,16 @@ void do_fetch(size_t num)
             err = 1;
             return;
         }
+        counts[num]++;
+        time = ossl_time_now();
     }
-    end = ossl_time_now();
-    times[num] = ossl_time_subtract(end, start);
+    while (time.t < max_time.t);
```

Comment:
style nit: can you move the `while()` statement so it follows the block closing bracket '}' at the same line? thanks.